static/js/people.js

```diff
@@ -1034,13 +1036,20 @@ exports._add_user = function add(person) {
 exports.add = function (person) {
     active_user_dict.set(person.user_id, person);
     exports._add_user(person);
+    non_active_user_dict.delete(person.user_id);
+};
```

Comment:
Should this check if `person.is_active` before deciding how to add the user?  I can imagine us getting an event from the server for a new, deactivated user, in some future code path.

Reply:
hmm but we don't have the `is_active` in the frontend. It's crippled on the backend.

Reply:
Maybe we should rename `.add()` to `.add_active_user()`.

Reply:
yes, I was thinking about that too. To the other commit where I plan to edit two existing functions.

Reply:
Here's the relevant calling code for context, @timabbott:

~~~ js
244     case 'realm_user':
245         if (event.op === 'add') {
246             people.add(event.person);
247         } else if (event.op === 'remove') {
248             people.deactivate(event.person);
249             stream_events.remove_deactivated_user_from_all_streams(event.person.user_id);
250         }
~~~

I think the `is_active` is implicit from `event.op` here, so the dispatch code is responsible for dispatching to the correct function.  So I support a rename here.  This could possibly be a prep commit before the bigger refactor from @clarammdantas.